ui/Screen/Org/Projects.svelte

```diff
@@ -41,20 +57,65 @@
     max-width: var(--content-max-width);
     min-width: var(--content-min-width);
   }
+
+  .pending {
+    margin-bottom: 2rem;
+  }
+
+  .header {
+    display: flex;
+    padding: 0.5rem 3rem 0.5rem;
+    width: 100%;
+  }
 </style>
 
 <div class="container">
-  {#if anchoredProjects.length !== 0 || unresolvedAnchors.length !== 0}
-    <ProjectList
-      projects={anchoredProjects}
-      userUrn={session.identity.urn}
-      on:select={select} />
-
-    <UnresolvedAnchorList anchors={unresolvedAnchors} />
+  {#if pendingResolved.length !== 0 || pendingUnresolved.length !== 0}
+    <div class="pending">
+      <div class="header">
+        <p class="typo-text-bold">Pending</p>
+        <p style="margin-left: .5rem; color: var(--color-foreground-level-6);">
+          Not enough of the members have signed this anchor.
+        </p>
+      </div>
+      {#if pendingResolved.length !== 0}
```

Comment:
We don't have to wrap lists in conditionals. If there are no elements in the array, the list won't be rendered. Fixed in 5518ed210d262ff6090d31f4369114c76e471e3a.